runtime/tests/checker/range_value_test.go

```diff
@@ -367,6 +367,13 @@ func TestCheckInclusiveRangeConstructionInvalid(t *testing.T) {
 		t,
 		"without_inner_type_annotation",
 		"let r: InclusiveRange<> = InclusiveRange(1, 10)",
-		[]error{&sema.InvalidTypeArgumentCountError{}},
+		[]error{&sema.InvalidTypeArgumentCountError{}, &sema.MissingTypeArgumentError{}},
+	)
+
+	runInvalidCase(
+		t,
+		"without_inner_type_annotation",
+		"let r: InclusiveRange = InclusiveRange(1, 10)",
```

Comment:
Can you please also add a general / top-level checker test case for testing `Checker.checkParameterizedTypeIsInstantiated`? 
That way we don't miss testing this general feature if we ever make changes to the range types/API.

Reply:
Added in 8732e3dee.